src/Raven.Studio/typescript/models/database/timeSeries/timeSeriesEntryModel.ts

```diff
@@ -0,0 +1,204 @@
+import timeSeriesValue  = require("models/database/timeSeries/timeSeriesValue");
+import generalUtils = require("common/generalUtils");
+
+class rollupDataModel {
+    first: timeSeriesValue;
+    last: timeSeriesValue;
+    min: timeSeriesValue;
+    max: timeSeriesValue;
+    sum: timeSeriesValue;
+    count = ko.observable<number>();
+
+    validationGroup: KnockoutValidationGroup;
+    
+    constructor (...values: number[]) {
```

Comment:
do we need vargs here? + remove space before `(`

Reply:
done